### ISSUE TITLE
Support RPG Maker 2000 v1.60 and v1.61

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#588]
I don't think it's worth to differentiate between 1.60 and 1.61
No idea how to detect them properly.

https://docs.google.com/document/d/1n-CHoQ0hwHWEnkoiLo7fiRUqS4hY4mhOkv_jkmz2cg8/edit?pli=1#heading=h.wcb2puxulvzi

v1.61 (2015-09-15)

Placeholders in system messages.
- [x] Battle messages can now contain placeholders (%S, %O, %V, and %U for subject, object, value and unit) instead of having a fixed order and position of those values.
- [x] Also applies to skill usage messages, state messages and inn messages.
- [x] additional values can be used in battle strings, such as the attacker’s name for evasion messages, the target name for skill messages (only for single-target skills), etc.
- [x] Battle messages which become too long for a line will automatically be word-wrapped.

v1.60
- [x] Picture operations (Show, Move, Erase) are not blocked by messages anymore.

**Comment by Ghabry** [comment]
inn_a_greeting_2 and inn_b_greeting_2 are now empty and merged into _1.

The rest is just adding of place holders

A new global chunk was added to the LDB 0x1A (defaults to 0). When this chunk is 1 the Database was updated to use placeholders.
So this is perfect for detecting RPG Maker 2k v1.61 and newer :)

**Comment by rohkea** [comment]
The messages about winning can contain codes (like ``\N[x]`` or ``\C[x]``).

When word-wrapping, RPG Maker 2000 v. 1.61 honours the codes that replace values (such as ``\N[0]``). However, the codes that do text formatting (such as ``\C[x]``) are counted as if they do take some screen estate:

![formatting-codes-when-word-wrapping](https://cloud.githubusercontent.com/assets/3201243/25075303/800731f8-231a-11e7-9739-39b3df91cf79.png)

The left screenshot has more formatting codes, so the first line is wrapped earlier than in the right screenshot.

Should we emulate this behaviour? Or should we do the right thing and consider that codes like ``\C[x]`` don’t take any place?

I can imagine situations when someone could depend on ``\C[x]`` taking screen estate when word-wrapping. People might use spaces get exact alignment of two lines, like this (a 'Hello world' is broken into two lines with spaces):

![exact-line-alignment](https://cloud.githubusercontent.com/assets/3201243/25075393/50577f42-231c-11e7-877c-27ddd5689b60.png)

But I really, really doubt anyone would do this. 😕

**Comment by Ghabry** [comment]
I just hope that nobody cared about using this for formating and would assume that \C[x] takes 0 space.

@CherryDT is that by design or a bug?

**Comment by CherryDT** [comment]
@Ghabry it's a bug in fact, thanks for pointing that out!
